compiler/varpartitions_symbol.nim

```diff
@@ -0,0 +1,1003 @@
+#
+#
+#           The Nim Compiler
+#        (c) Copyright 2020 Andreas Rumpf
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## Partition variables into different graphs. Used for
+## Nim's write tracking and also for the cursor inference.
+## The algorithm is a reinvention / variation of Steensgaard's
+## algorithm.
+## The used data structure is "union find" with path compression.
+
+import ast, types, lineinfos, options, msgs, renderer
+from trees import getMagic, whichPragma
+from wordrecg import wNoSideEffect
+from isolation_check import canAlias
+<<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
seriously? ;-)

Reply:
ugh I forgot to make sure it's removed: I rm-ed it but probably i add-ed again